cmd/minishift/cmd/start_preflight.go

```diff
@@ -433,22 +439,22 @@ func checkHypervDriverSwitch() bool {
 
 // checkHypervDriverInstalled returns true if Hyper-V driver is installed
 func checkHypervDriverInstalled() bool {
-	// Check if Hyper-V's Virtual Machine Management Service is installed
-	_, err := exec.LookPath("vmms.exe")
-	if err != nil {
-		return false
-	}
-
-	// check to see if a hypervisor is present. if hyper-v is installed and enabled,
 	posh := powershell.New()
 
+	// check to see if a hypervisor is present. if hyper-v is installed and enabled,
 	checkHypervisorPresent := `@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent`
-
 	stdOut, _, _ := posh.Execute(checkHypervisorPresent)
 	if !strings.Contains(stdOut, "True") {
 		return false
 	}
 
+	// Check if Hyper-V's Virtual Machine Management Service is running
+	checkVmmsRunning := `@(Get-Service vmms).Status`
+	stdOut, _, _ = posh.Execute(checkVmmsRunning)
+	if strings.TrimSpace(stdOut) != "Running" {
```

Comment:
This might lead to a localization problem... can we test this?

Reply:
I believe I forced the powershell session to be unaffected by localization.... or perhaps this was just a single test for the default switch name? Please check...

Reply:
Nice catch.. need to check this, in case of the default switch, we used the ID instead of the name to get around the localization problem

Reply:
right... but I also remember I added a check to docker/machine that sets the locale to work around their issue (as they use the name). I'll check the code...

Reply:
https://github.com/docker/machine/commit/5bc065d6eb72a0fccaf04e162ff3e723b7dc5b09

Oops... this sets the encoding output... but I am sure this can also be used for the locale. ;-)